index.js

```diff
@@ -17,7 +17,7 @@ program
   .description('List Linda Ikeji Top Stories')
   .option('-n, --number <int>", "specify number of stories')
   .action(function(options){
-    var count = (typeof options === 'object') ? 20 : options;
+    var count = (typeof options === 'object' || isNaN(parseInt(options)) || parseInt(options) == 0) ? 20 : options;
     console.log("List top " + count + " Linda Ikeji Stories");
```

Comment:
Damn...why do we have to compare three items like this?. I think we can just remove `typeof options === 'object' since we are using `isNaN` already like so:

```js
var count = (isNaN(options) || parseInt(options) == 0) ? 20 : options;
```
Please fix. Thanks